### PR TITLE
Add copyrights to Dockerfiles

### PR DESCRIPTION
**Pull request by aneshlya**

### CODE REVIEW

File: docker/centos/7/xpu_ispc_build/Dockerfile [hunk 1]
```diff
@@ -46,6 +46,11 @@ ENV XE_DEPS=/usr/local/deps
 # http://www.intel.com/software/sde and download the latest version,
 # extract it, add to path and set SDE_HOME.
 
+#
```
Comment (dbabokin) [inline docker/centos/7/xpu_ispc_build/Dockerfile:49]: Supposed to be in the beginning of the file

File: docker/centos/8/cpu_ispc_build/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,9 @@
+
```
Comment (dbabokin) [inline docker/centos/8/cpu_ispc_build/Dockerfile:1]: Empty first line here and many others is really not needed.

To my taste "Copyright" should be on the very first line, but one line with empty comment is ok as well.

File: docker/centos/7/xpu_ispc_build/Dockerfile (hunk 1 above)
Comment (aneshlya) [reply docker/centos/7/xpu_ispc_build/Dockerfile:49]: I have no idea how it appeared here :)

**Review by dbabokin — APPROVED** [review]
LGTM!

**Pull request merged**